### PR TITLE
amp-analytics support SensorsData

### PR DESCRIPTION
**Pull request by 869359954**
# ♿ amp-analytics support SensorsData

# description
- Sensors Data is a Third party analysis service provider
- we want amp-analytics to support SensorsData

### CODE REVIEW

File: examples/analytics-vendors.amp.html [hunk 1]
```diff
@@ -45,6 +45,7 @@
     <label for="filter">Vendor filter</label>
     <select id="filter" name="type">
       <option>[Select a vendor]</option>
+      <option>sensorsanalytics</option>
```
Comment (micajuine-ho) [inline examples/analytics-vendors.amp.html:48]: Alphabetical order please.

File: examples/analytics-vendors.amp.html [hunk 2]
```diff
@@ -136,6 +137,49 @@
...
+ <!-- sensorsanalytics stats tracking -->
+
+<amp-analytics type="sensorsanalytics" id="sensoranalytis1">
+  <script type="application/json">
+      
+      {
+        
+        "requests": {
+          "event": "https://jssdkdata.debugbox.sensorsdata.cn/sa?project=liangshuang"
+        },
```
Comment (micajuine-ho) [inline examples/analytics-vendors.amp.html:149]: Place this chunk in alphabetical order with the other vendor chunks.

File: examples/analytics-vendors.amp.html [hunk 3]
```diff
@@ -136,6 +137,49 @@
 <div id="container">
 Container for analytics tags. Positioned far away from top to make sure that doesn't matter.
 
+ <!-- sensorsanalytics stats tracking -->
+
```
Comment (micajuine-ho) [inline examples/analytics-vendors.amp.html:141]: nit: remove newline

File: examples/analytics-vendors.amp.html [hunk 4]
```diff
@@ -136,6 +137,49 @@
...
+<amp-analytics type="sensorsanalytics" id="sensoranalytis1">
+  <script type="application/json">
+      
+      {
+
```
Comment (micajuine-ho) [inline examples/analytics-vendors.amp.html:146]: nit: remove newlines

File: extensions/amp-analytics/0.1/vendors/sensorsanalytics.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
+
```
Comment (micajuine-ho) [inline extensions/amp-analytics/0.1/vendors/sensorsanalytics.json:1]: remove newline

**Review by micajuine-ho — CHANGES_REQUESTED** [review]

**Comment by 869359954** [comment]
@micajuine-ho  I have fixed the previous problem, please help help help me review

**Review by micajuine-ho — CHANGES_REQUESTED** [review]
You need to add tests to `extensions/amp-analytics/0.1/test/vendor-requests.json` based off of `extensions/amp-analytics/0.1/vendors/sensorsanalytics.json`.

**Comment by 869359954** [comment]
@micajuine-ho  I got it !  I have added . Please help help help me review  ,thank you

**Review by micajuine-ho — APPROVED** [review]

**Pull request merged**